lib/network/kmip/Client.ts

```diff
@@ -339,7 +339,8 @@ export default class Client implements KMSInterface {
                 kmipMsg(operation, keyIdentifier,
                     'Server did not return the expected identifier')
             );
-            logger.error(`KMIP::${operation}`,
+            // warn level to avoid dumping debug and trace logs on retryable errors
```

Comment:
so do we not consider these as errors, seems like an unhealthy server and valid error even if we can retry.
I am not asking to change this back, but trying to understand more as this is not just about debug and trace logs but also about the error log file.

Reply:
When testing I see it dumps unrelated logs in debug and trace (mostly related to auth) and we have no debug or traces related to KMIP. The log can also be self contained, it has the operation and keys that differ.

And in cluster mode, with retry this become even less critical as another host can succeed on the request.

So it helps avoid many other unrelated logs, even for a simple non HA cluster kmip, we don't need to bump the debug logs.

I see we almost never use this `warn` level.

Reply:
yes we do not use warn as it adds to normal s3 logs and we want to keep normal logs as lean as possible. If its an error it should go to error logs.

Reply:
but those error logs go in the same log file

Reply:
[Discussed over slack](https://scality.slack.com/archives/C01N31LU4HW/p1747748084977989?thread_ts=1747736902.944369&cid=C01N31LU4HW)